test/e2e/smoke-test/packages/contour/1.18.2/contour-test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+
+# Checking package is installed or not
+tanzu package installed list | grep "contour.community.tanzu.vmware.com"
+exitcode_contour=$?
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "${MY_DIR}"
+if [ "${exitcode_contour}" == 1 ] 
+then
+    version=$(tanzu package available list contour.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install contour --package-name contour.community.tanzu.vmware.com -f "${MY_DIR}"/contour-values.yaml --version "${version}"
+fi
+sleep 15s
+
+# Providing prerequisite 
+kubectl create namespace contour-example-workload
```

Comment:
I think this test follows the idea of a separate namespace like mentioned in https://github.com/vmware-tanzu/community-edition/pull/2246/files?authenticity_token=450WwLofW6a7J2Faj%2BDF9zLMPoeRGfikVXHqHGJQ2UNAYdPOmDh0aCxdnGOpjeP3aEufBmSe2Jsw8THsAqKXGA%3D%3D&file-filters%5B%5D=.sh&file-filters%5B%5D=.yaml#r741061685 . I think we could also randomize this with to reduce any possibility of clashing resource name issues

```bash
kubectl create namespace contour-example-workload-${RANDOM}
```

or even shorten the name like

```bash
kubectl create namespace contour-test-${RANDOM}
```